gbdxtools/images/meta.py

```diff
@@ -460,6 +461,17 @@ def _rgb_bands(self):
     def _ndvi_bands(self):
         return [6, 4]
 
+    def base_layer_match(self, **kwargs):
+        rgb = self.rgb(**kwargs)
+        if "blm" in kwargs and not kwargs["blm"]:
+            return rgb
+        from gbdxtools.images.tms_image import TmsImage
+        tms = TmsImage(zoom=18, bbox=self.bounds, **kwargs)
```

Comment:
Since the base layer matching behavior is default when available we might want to select a zoom level dynamically to avoid pulling large amounts of data.

Reply:
Also self.bounds is not necessarily in EPSG:4326

Reply:
what do you think the best way to select the zoom would be? I'll reproject the bounds to 4326 too.